src/tokens.js

```diff
@@ -271,25 +271,23 @@ const tokens = {
   createToken(input, output, address, name, symbol, mintAmount, pin) {
     // Create authority output
     // First the tokens masks that will be the value for the authority output
-    const tokenMasks = TOKEN_CREATION_MASK | TOKEN_MINT_MASK | TOKEN_MELT_MASK;
-    // Create token uid
-    const tokenUID = this.getTokenUID(input.tx_id, input.index);
-    const authorityOutput = {'address': address, 'value': tokenMasks, 'tokenData': AUTHORITY_TOKEN_DATA};
+    const tokenMasks = TOKEN_CREATION_MASK;
```

Comment:
I think we should include the `TOKEN_MINT_MASK | TOKEN_MELT_MASK` flags. But we should discuss it in the RFC, then we change here according to the decision there.